setup.py

```diff
@@ -18,14 +18,17 @@ def get_readme_md_contents():
 with open("datadog/version.py") as fp:
     exec(fp.read(), version)
 
-install_reqs = ["decorator>=3.3.2", "requests>=2.6.0"]
+# `typing` package is only required for Python versions older than 3.5, but we include it here so
+# that a single wheel build can be used for all supported Python versions. Installing `typing` on
+# Python 3.5+ has no effect.
+#
+# `configparser` package is only required for Python versions older than 3 (it is included here for
+# the same reason as the `typing` package).
+install_reqs = ["decorator>=3.3.2", "requests>=2.6.0", "typing", "configparser"]
```

Comment:
Would following code work (see https://www.python.org/dev/peps/pep-0508/)?
```suggestion
install_reqs = [
    "decorator>=3.3.2",
    "requests>=2.6.0",
    'typing;python_version<"3.5"',
    'configparser;python_version<"3.0"',
]
```

Reply:
I tested, and it does. Commiting.

Reply:
Thanks @jirikuncar and @zippolyte, I wasn't aware of Python version specific dependencies in `install_requires`. This is a much better fix! 🎉